e2e/capacity/list_unclaimed_rewards.test.ts

```diff
@@ -0,0 +1,72 @@
+import '@frequency-chain/api-augment';
+import assert from 'assert';
+import { ExtrinsicHelper } from '../scaffolding/extrinsicHelpers';
+import { getFundingSource } from '../scaffolding/funding';
+import {
+  createKeys,
+  createMsaAndProvider,
+  DOLLARS,
+  createAndFundKeypair,
+  boostProvider,
+  getNextRewardEraBlock,
+} from '../scaffolding/helpers';
+import { isTestnet } from '../scaffolding/env';
+import { KeyringPair } from '@polkadot/keyring/types';
+
+const fundingSource = getFundingSource('capacity-list-unclaimed-rewards');
+
+describe('Capacity: list_unclaimed_rewards', function () {
+  const providerBalance = 2n * DOLLARS;
+
+  const setUpForBoosting = async (boosterName: string, providerName: string): Promise<[number, KeyringPair]> => {
+    const booster = await createAndFundKeypair(fundingSource, 5n * DOLLARS, boosterName);
+    const providerKeys = createKeys(providerName);
+    const provider = await createMsaAndProvider(fundingSource, providerKeys, providerName, providerBalance);
+    await assert.doesNotReject(boostProvider(fundingSource, booster, provider, 1n * DOLLARS));
+
+    return [provider.toNumber(), booster];
+  };
+
+  it('can be called', async function () {
+    const [_provider, booster] = await setUpForBoosting('booster1', 'provider1');
+    const result = await ExtrinsicHelper.apiPromise.call.capacityRuntimeApi.listUnclaimedRewards(booster.address);
+    assert.equal(result.length, 0, `result should have been empty but had ${result.length} items`);
+  });
+
+  it('returns correct rewards after enough eras have passed', async function () {
+    if (isTestnet()) {
+      this.skip();
+    }
+    const [_provider, booster] = await setUpForBoosting('booster2', 'provider2');
+    console.debug(`Booster pubkey: ${booster.address}`);
+
+    // Move out of the era we boosted inside of
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+    // Have three eras where we get rewards
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+
+    const result = await ExtrinsicHelper.apiPromise.call.capacityRuntimeApi.listUnclaimedRewards(booster.address);
+
+    assert.equal(result.length, 4);
+
+    // This is the era we first boosted in, shouldn't have any rewards
+    assert.equal(result[0].staked_amount.toHuman(), '1.0000 UNIT');
+    assert.equal(result[0].eligible_amount.toHuman(), '0');
+    assert.equal(result[0].earned_amount.toHuman(), '0');
```

Comment:
@shannonwells, I updated this to match the rust test as this was failing with what I thought was an issue with a fence post, but now I am not so sure. Sometimes we get this result, and sometimes we do not? Thoughts as to why?

Reply:
Found and fixed. It was a Genesis config issue

Reply:
Note: we've fixed the issue with this test by adding a Genesis config.